pkg/util/cipher/cipher.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cipher
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/component-base/cli/flag"
+)
+
+var tlsVersionMap = map[string]uint16{
+	"VersionTLS10": tls.VersionTLS10,
+	"VersionTLS11": tls.VersionTLS11,
+	"VersionTLS12": tls.VersionTLS12,
+	"VersionTLS13": tls.VersionTLS13,
+}
+
+// NewTLSConfig generates a *tls.Config with cipher suites string, TLS min version and TLS max version.
```

Comment:
s/with/given

Reply:
Updated to `tls.Config with given cipher suites ...`
Not sure if this is your intention or `tls.Config given cipher suite ...` but it sounds better for me.